topics/event-classes.md

```diff
@@ -6,32 +6,70 @@ body_class: event-classes
+| `player.view` | A resource was opened in the player by a student. `digitisation.view` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.view`. |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.view.contributors` | A resource was opened in the player by a contributor (academic, teaching assistant) |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-view-contributors)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createannotation` | A new annotation (comment thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createannotation)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.annotate.createreply` | A new replay (to an existing thread) was created by a user |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-annotate-createreply)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.profile.complete` | A user completed their profile |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-profile-complete)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.search` | A user used the search function |  [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-search)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+| `player.download` | The player resource was downloaded by a user. `digitisation.download` is also recorded by the Talis Elevate player when the content has originated from Copyright Clearance. This is in addition to `player.download`. | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#player-download)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
+|`digitisation.print`| A digitisation played through talis elevate was accessed for printing. This event is only emitted when a digitisation is played in Talis Elevate. | [<i class="yes fas fa-check-circle"><span class="sr-only">Available</span></i>](timeseries.html#digitisation-print)  |  <i class="no fas fa-times-circle"><span class="sr-only">Not available</span></i>  |
```

Comment:
I guess we should probably add `player.print` and the other missing player events to the config... but one for a follow up